### PR TITLE
Add docs, detailing how to install CNI with Helm

### LINKED ISSUE #602: Add docs for installing CNI chart
Update the following docs to show how to use the CNI chart:

* [installing Linkerd with Helm](https://linkerd.io/2/tasks/install-helm/)
* [CNI plugin](https://linkerd.io/2/features/cni/) (just a reference to the above doc should suffice)

E.g. on GKE,
```
$ helm repo update

$ helm search linkerd-cni
NAME                       CHART VERSION    APP VERSION     DESCRIPTION
linkerd-edge/linkerd2-cni   19.12.1         edge-19.12.1    Linkerd gives you observability, reliability, and securit...
linkerd/linkerd2-cni        2.7.0           stable-2.7.0    Linkerd gives you observability, reliability, and securit...

$ helm install --name=linkerd2-cni --set DestCNIBinDir=/home/kubernetes/bin charts/linkerd2-cni
$ helm install --name=linkerd2 --set Identity.Issuer.CrtExpiry="Dec 9 16:10:37 2019" --set-file Identity.TrustAnchorsPEM=ca.crt --set-file Identity.Issuer.TLS.CrtPEM=issuer.crt --set-file Identity.Issuer.TLS.KeyPEM=issuer.key --set NoInitContainer=true --set InstallNamespace=false charts/linkerd2
```

### PR DESCRIPTION
**Pull request by zaharidichev**
Modifies docs to include instructions of how to install CNI with Helm.

Fixes #602

Signed-off-by: Zahari Dichev <zaharidichev@gmail.com>

### CODE REVIEW

File: linkerd.io/content/2/features/cni.md [hunk 1]
```diff
@@ -62,3 +62,8 @@ The most important flags are:
    logs. One way to do this is to log onto the node and use
    `journalctl -t kubelet`. The string `linkerd-cni:` can be used as a search to
    find the plugin log output.
+
+{{< note >}}
+If you would like to Linkerd with CNI support using Helm, you can read how to
```
Comment (grampelberg) [inline linkerd.io/content/2/features/cni.md]: WDYT about moving this all out into an installing cni section that includes both methods?

Comment (zaharidichev) [reply linkerd.io/content/2/features/cni.md]: Makes sense.

File: linkerd.io/content/2/tasks/install-helm.md [hunk 2]
```diff
@@ -38,10 +38,10 @@ exp=$(date -d '+8760 hour' +"%Y-%m-%dT%H:%M:%SZ")
 
 helm install \
   --name=linkerd2 \
-  --set-file Identity.TrustAnchorsPEM=ca.crt \
-  --set-file Identity.Issuer.TLS.CrtPEM=issuer.crt \
-  --set-file Identity.Issuer.TLS.KeyPEM=issuer.key \
-  --set Identity.Issuer.CrtExpiry=$exp \
+  --set-file global.identity.trustAnchorsPEM=ca.crt \
```
Comment (ihcsim) [inline linkerd.io/content/2/tasks/install-helm.md]: Per https://github.com/linkerd/website/pull/627#discussion_r365299706, these `identity` variables aren't `global`.

File: linkerd.io/content/2/features/cni.md [hunk 3]
```diff
@@ -42,6 +44,38 @@ linkerd install --linkerd-cni-enabled | kubectl apply -f -
...
+to generate new ones.
+
+```bash
+# install the CNI plugin first
+helm install --name=linkerd2-cni linkerd2/linkerd2-cni
+# ensure the plugin is installed and ready
+linkerd check --pre --linkerd-cni-enabled
+```
+
+At that point you are ready to install Linkerd with CNI enabled:
```
Comment (grampelberg) [inline linkerd.io/content/2/features/cni.md]: Let's add a link to the helm install docs and just mention the value that is required. I'd rather not duplicate the instructions if at all possible.

File: linkerd.io/content/2/features/cni.md [hunk 4]
```diff
@@ -42,6 +44,38 @@ linkerd install --linkerd-cni-enabled | kubectl apply -f -
 This will set a `cniEnabled` flag in the global `linkerd-config` ConfigMap. All
 subsequent proxy injections will read this field and omit init containers.
 
+### Using Helm
+
+Before you begin, you need to have TLS certificates. You can
+provide your own, or follow [these instructions](/2/tasks/generate-certificates/)
+to generate new ones.
+
+```bash
```
Comment (ihcsim) [inline linkerd.io/content/2/features/cni.md]: Maybe add a separate snippet to remind users to update their Helm local cache. Something like:

Ensure that your Helm local cache is updated:
```
helm repo update 

helm search linkerd2-cni                                                   
NAME                      CHART VERSION APP VERSION DESCRIPTION
linkerd-edge/linkerd2-cni   20.1.1        edge-20.1.1 A helm chart containing the resources needed by the Linke...
linkerd-stable/linkerd2-cni  2.7.0        stable-2.7.0 A helm chart containing the resources needed by the Linke...
```

Run the following commands to install the CNI DaemonSet:
```
# install the CNI plugin first
helm install --name=linkerd2-cni linkerd2/linkerd2-cni

# ensure the plugin is installed and ready
linkerd check --pre --linkerd-cni-enabled
```

File: linkerd.io/content/2/features/cni.md [hunk 5]
```diff
@@ -42,6 +44,36 @@ linkerd install --linkerd-cni-enabled | kubectl apply -f -
...
+# install the CNI plugin first
+helm install --name=linkerd2-cni linkerd2/linkerd2-cni
+
+# ensure the plugin is installed and ready
+linkerd check --pre --linkerd-cni-enabled
+```
+
+At that point you are ready to install Linkerd with CNI enabled.
+You can follow [Installing Linkerd with Helm](/2/tasks/install-helm/) to do so.
```
Comment (alpeb) [inline linkerd.io/content/2/features/cni.md]: You could add here a note pointing out that `--set global.noInitContainer=true --set installNamespace=false` flags are required.

Comment (ihcsim) [reply linkerd.io/content/2/features/cni.md]: What do you think of adding this to the _Installing Linkerd with Helm_ doc, under a new subsection called _Disabling The Proxy Init Container_, before the _Setting High Availability_ subsection? It feels a bit weird to refer the reader to a different page, follow by another note about installation below.

File: linkerd.io/content/2/features/cni.md [hunk 6]
```diff
@@ -42,6 +44,36 @@ linkerd install --linkerd-cni-enabled | kubectl apply -f -
 This will set a `cniEnabled` flag in the global `linkerd-config` ConfigMap. All
 subsequent proxy injections will read this field and omit init containers.
 
+### Using Helm
+
+Before you begin, you need to have TLS certificates. You can
```
Comment (ihcsim) [inline linkerd.io/content/2/features/cni.md]: This is no longer needed since we are now referring the readers to the _Installing with Helm_ page.

**Pull request merged**